docs/getting-started/create-comit-app.md

```diff
@@ -4,6 +4,286 @@ title: Getting Started
 sidebar_label: Create-comit-app
 ---
 
-We are working extensively on pushing out a new documentation. If you want to help out, feel free to grab a ticker from [here](https://github.com/comit-network/comit-network.github.io) :)
+[`create-comit-app`](https://github.com/comit-network/create-comit-app/) is a project to help COMIT App developers setup an environment.
 
-In the meantime, if you want to get started, checkout our [create-comit-app](https://github.com/comit-network/create-comit-app/) and the provided [examples](https://github.com/comit-network/create-comit-app/tree/master/create/new_project).
+While mainly written in Rust, it is actually shipped on [npmjs.com](https://npmjs.com) as two packages:
+- [`create-comit-app`](https://www.npmjs.com/package/create-comit-app)
+- [`comit-scripts`](https://www.npmjs.com/package/comit-scripts)
+
+While you can build your app in JavaScript, examples and the [SDK](TODO) are written in TypeScript,
+allowing you to also build in TypeScript.
+
+To use either packages, you need to have [`yarn`](https://yarnpkg.com/) or [`npm`](https://www.npmjs.com/get-npm) installed.
+In this tutorial we will use `yarn` but it can be replaced by `npm` in any command below. 
+
+## TL;DR
+
+```
+yarn create comit-app my-app
+cd my-app/demos/btc_eth
+yarn install
+yarn run start-env
+```
+Now, open a new terminal (keep `start-env` running):
+
+```
+cd my-app/demos/btc_eth
+yarn run swap
+```
+
+Congratulations, you ran an atomic swap!
+
+## The long way
```

Comment:
I think this section needs more WHY answers - at the moment it is only WHAT. E.g. we state that we start 2 cnd nodes, but we don't state why. 
I think we need a bit of an introduction of what is ongoing and why something is done.

Inspiration: https://0x.org/docs/guides/swap-tokens-with-0x-api

I don't have the perfect recipe for this yet. I thought it might help to tell a story. But that might make it too long.